config.json

```diff
@@ -114,6 +114,7 @@
         "nougat",
         "inist",
         "istex",
+        "istex-Data&Lodex",
```

Comment:
Avoid uppercase and special character in theme id

Reply:
ok je modifie ça, merci

Reply:
@AlasDiablo c'est fait